R/bpmnVisualization.R

```diff
@@ -26,25 +26,25 @@
 #'      Use an explicit element ID for the widget (rather than an automatically
 #'      generated one). Useful if you have other JavaScript that needs to explicitly
 #'      discover and interact with a specific widget instance.
-#' 
-#' @returns A \code{bpmn-visualization} Widget that will intelligently print itself into HTML in a variety of contexts 
+#'
+#' @returns A \code{bpmn-visualization} Widget that will intelligently print itself into HTML in a variety of contexts
 #'      including the R console, within R Markdown documents, and within Shiny output bindings.
-#'      
-#' @examples 
+#'
+#' @examples
 #' # Load the BPMN file
 #' bpmn_file <- system.file("examples/Order_Management.bpmn", package = "bpmnVisualization")
-#' 
+#'
 #' # Display the BPMN diagram
-#' display(bpmn_file, width='auto', height='auto')
-#' 
+#' bpmnVisualization::display(bpmn_file, width='auto', height='auto')
```

Comment:
In my opinion, adding package name as prefix make more complicated to understand the important information here (the method name & the parameter).
If it's important, we can add another example to explain how to use with the package name 🙂

Reply:
Finally, after discussion with @tbouffard, we decided to merge this PR like it is.